docs/pages/technical/registry.rst

```diff
@@ -13,13 +13,16 @@ An event is structured as follows:
     Event:
       IssuedAt (timestamp)
       Type (string enum)
+      Version (int)
+      Ref (hex SHA-1 hash)
+      Prev (hex SHA-1 hash)
       Signature (JWS)
       Payload (object)
 
-It contains 2 sections, its headers (*IssuedAt*, *Type* and *Signature*) and a human-readable copy of the content of event (*Payload*).
-We call it a copy because the authenticated content of the event is found encoded inside the *Signature* field. The *Payload*
+It contains 2 sections, its headers (``IssuedAt``, ``Type``, ``Signature``, ``Version``, ``Ref``, ``Prev``) and a human-readable copy of the content of event (``Payload``).
```

Comment:
What about a description on what the new fields are for? (except version, which is explained)

Reply:
Added.